scripts/zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,588 @@
+
+
+def value_bytes(groups: dict[str, Any]) -> int:
+    return sum(_int(groups[group], group) for group in VALUE_GROUPS)
+
+
+def row_by_path(accounting: dict[str, Any], relative_path: str) -> dict[str, Any]:
+    for row in _list(accounting.get("rows"), "accounting.rows"):
+        row_obj = _dict(row, "accounting row")
+        if row_obj.get("evidence_relative_path") == relative_path:
+            return row_obj
+    raise RmsnormLabelSensitivityError(f"missing accounting row: {relative_path}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)